### ISSUE TITLE
[tr064] Call list has incorrect time when call between 12:00 and 13:00

### ISSUE THREAD

**Issue by sidamos** [openhab/openhab-addons#10162]
## Expected Behavior
A call at 12:12 in the afternoon should show as 12:12 in the call list.

## Current Behavior
A call at 12:12 in the afternoon shows as 00:12 in the call list. Calls after 13:00 seem to be OK.

## Steps to Reproduce (for Bugs)
1. Make a call between 12:00 and 13:00 in the afternoon
2. Fetch call list

* Version used: OH 3.0.1, TR064 3.0.1
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...): Java 11
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...): Linux

```
[
  {
    "localNumber": "xxx",
    "remoteNumber": "xxx",
    "date": "2021-02-14T17:06:00+01",
    "type": 3,
    "duration": 1
  },
  {
    "localNumber": "xxx",
    "remoteNumber": "xxx",
    "date": "2021-02-14T00:12:00+01",
    "type": 2,
    "duration": 0
  },
  {
    "localNumber": "xxx",
    "remoteNumber": "xxx",
    "date": "2021-02-12T15:39:00+01",
    "type": 2,
    "duration": 0
  },
```

![fritzbox](https://user-images.githubusercontent.com/873537/107918623-50ef2b00-6f6a-11eb-92d0-9671755f50d2.png)

**Comment by hare0815** [comment]
I can confirm this problem.

Version used: 3.1.0-SNAPSHOT (#2207)

openhab.log  with TRACE:  (correct time in the call data: 12:20)

```
2021-02-15 12:20:28.865 [TRACE] [hab.binding.tr064.internal.util.Util] - Refreshing cache for 'https://192.168.200.100:49443/calllist.lua?sid=3f19dd2a0683afb9&days=1'
2021-02-15 12:20:29.431 [TRACE] [hab.binding.tr064.internal.util.Util] - XML = <?xml version="1.0" encoding="UTF-8"?> 
<root>
<!-- days:1 -->
<!-- max:999 -->
<!-- id:nil -->
<!-- timestamp:nil -->
<!-- calls:400 -->
<timestamp>1385922161</timestamp>
<Call><Id>18645</Id><Type>1</Type><Caller>0156nnnnnn2</Caller><Called>ITD</Called><CalledNumber>9nnnnn0</CalledNumber><Name>hh satellite</Name><Numbertype>sip</Numbertype><Device>MT-D</Device><Port>11</Port><Date>15.02.21 12:20</Date><Duration>0:01</Duration><Count></Count><Path /></Call>
...
<!-- calls:0 -->
</root>

2021-02-15 12:20:29.550 [TRACE] [hab.binding.tr064.internal.util.Util] - Storing in cache org.openhab.binding.tr064.internal.dto.additions.Root@154839f
2021-02-15 12:20:29.554 [TRACE] [hab.binding.tr064.internal.util.Util] - Returning from cache: org.openhab.binding.tr064.internal.dto.additions.Root@154839f
```
events.log: (wrong time: 00:20)

```
2021-02-15 12:20:32.321 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'dtFritzBoxTR064_CallList1_date' changed from 2021-02-15T00:11:00.000+0100 to 2021-02-15T00:20:00.000+0100
```

**Comment by sidamos** [comment]
I updated to the snapshot binding a few days ago and the bug still exists.

**Comment by hare0815** [comment]
The error is still present for me in openHAB 3.1.0 build #2324.

**Comment by hare0815** [comment]
The error is solved for me in openHAB 3.1.0 build #2326.
